src/public/peoplePicker.ts

```diff
@@ -0,0 +1,97 @@
+import { ensureInitialized, isAPISupportedByPlatform } from '../internal/internalAPIs';
+import { FrameContexts } from './constants';
+import { ErrorCode, SdkError } from './interfaces';
+import { validatePeoplePickerInput } from '../internal/mediaUtil';
+import { sendMessageToParent } from '../internal/communication';
+
+export namespace peoplePicker {
+  /**
+   * This is the SDK version when people picker API is supported on mobile.
+   */
+  export const peoplePickerRequiredVersion = '2.0.0';
```

Comment:
We are actually going to v1.10.0 next. I noticed that we were using 2.0.0 here and also for some of the new media APIs. Any chance we can update that? Have we taken any dependencies on the 2.0.0 number for any in-production code yet?

Reply:
Can not we fix compareSDKVersions function and use 1.10.0. I am hesitating to go with 2.0.0 which indicate major version changes means breaking changes.

@SonalJha has any previous version of prod sdk release used 2.0.0 ? if not then definitely we should fix compareSDKVersions  and use 1.10.0 here.

Reply:
even if we fix compareSDKVersion() the older versions e.g. 1.9.0 will still break